wiki/Glossary/en.md

```diff
@@ -474,54 +459,55 @@ Thus mappers are advised to use it judiciously.
 Accumulated kudosu can be used to increase a beatmap's priority rating.
 Any mapper or modder may spend 1 kudosu to give any map a star, which will help the map in the approval process by giving it more star priority.
 
 Before awarding kudosu keep the following things in mind:
 
--     Did the post help you in any way **to improve your map**?
--     Did the modder post things **that AIMod cannot pick up**?
--     Did the post show that the modder has **knowledge of modding**?
--     Did the modder post things that generally **make sense to you and seem not too terribly picky?** If their post did seem picky, are you sure that the things pointed out **can improve your mapset anyways?**
--     Even if you **don't** follow the modder's advice, did you not follow it because of **your personal choice only**, and not because **the modding seemed pointless to you?**
--     Did you not already give the modder kudosu for a mod post on the same map **recently**, and if you did, has the map **vitally changed** between the two mod posts (adding a difficulty, remapping a difficulty)?
--     Was this a **substantial mod post?** Did the mod post point out more than a couple of things to improve or fix your map?
+- Did the post help you in any way **to improve your map**?
+- Did the modder post things **that AIMod cannot pick up**?
+- Did the post show that the modder has **knowledge of modding**?
+- Did the modder post things that generally **make sense to you and seem not too terribly picky?** If their post did seem picky, are you sure that the things pointed out **can improve your mapset anyways?**
+- Even if you **don't** follow the modder's advice, did you not follow it because of **your personal choice only**, and not because **the modding seemed pointless to you?**
+- Did you not already give the modder kudosu for a mod post on the same map **recently**, and if you did, has the map **vitally changed** between the two mod posts (adding a difficulty, remapping a difficulty)?
+- Was this a **substantial mod post?** Did the mod post point out more than a couple of things to improve or fix your map?
 
 If you can answer yes to all of these questions, **feel free to give the modder their well-earned kudosu!**
-All a mapper has to do to give a modder kudosu is simply go to the mod post and click the "Give Kudos" button that is located under the poster's information.
+
+The creators of a beatmap can give a modder kudosu by simply going to the mod post and click the `Give Kudos` button that is located under the poster's information.
 Please be aware that beatmap nominators may revoke your kudosu if they deem that your post should not have been awarded kudosu!
 
 ## L
 
 ### Lead-in
 
-The lead-in time is a 3 second time period of silence that might automatically be inserted before the beginning of a beatmap.
+The lead-in time is a ~3 second time period of silence that might automatically be inserted before the beginning of a beatmap.
```

Comment:
hm, actually, isn't it from 0 to 3 seconds? Depends on where the first hitobject is located, time-wise?